mipidsi/src/models/rm67162.rs

```diff
@@ -0,0 +1,76 @@
+use embedded_graphics_core::pixelcolor::Rgb565;
+use embedded_hal::delay::DelayNs;
+
+use crate::{
+    dcs::{
+        BitsPerPixel, ExitSleepMode, InterfaceExt, PixelFormat, SetAddressMode, SetDisplayOn,
+        SetInvertMode, SetPixelFormat,
+    },
+    interface::Interface,
+    options::ModelOptions,
+};
+
+use super::Model;
+
+/// RM67162 AMOLED display driver implementation
+/// Supports:
```

Comment:
There needs to be a new line between the first line and the rest of the text. Otherwise the struct list in the docs is broken:

![image](https://github.com/user-attachments/assets/b4b37365-6918-4656-a218-82818b8738db)


```suggestion
/// RM67162 AMOLED display driver implementation
///
/// Supports:
```